pkg/agent/controller/trafficcontrol/controller_test.go

```diff
@@ -288,7 +288,7 @@ func generateTrafficControlState(direction v1alpha2.Direction,
 }
 
 func waitEvents(t *testing.T, expectedEvents int, c *fakeController) {
-	require.NoError(t, wait.PollImmediate(10*time.Millisecond, 5*time.Second, func() (done bool, err error) {
+	require.NoError(t, wait.Poll(10*time.Millisecond, 5*time.Second, func() (done bool, err error) {
```

Comment:
why this change?

Reply:
to avoid expectedEvents=0 being returned too quickly (the event may haven't been handled yet)